subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/SwiftXCTestSuite.java

```diff
@@ -30,5 +31,12 @@
     /**
      * Returns the executable that is built to run this test suite.
      */
-    SwiftBinary getExecutable();
+    SwiftBundle getExecutable();
```

Comment:
**Open Questions:**
- For now, on macOS, the "executable" is a bundle. However, on Linux, this won't be true anymore. I'm unsure how to proceed forward. It may be correct for this PR, but we will need to discuss how to handle Linux's XCTest executable binary.

Reply:
We should get rid of the bundle and just build an executable on MacOS as well, so we can do parallel test execution, rich events, test filtering and so on.